### PR TITLE
#185517557 Bug fix implementing feedbacks from the predemo

### PR DESCRIPTION
**Pull request by nishimweprince**
## What does this PR do?

This PR applies the latest feedback from the predemo of 30th June, 2023.

## Description of Task to be completed?

- Redesign the wishlist page to enhance user experience and make it more intuitive.
- Add a wishlist icon (heart icon) in the navbar for easy navigation to the wishlist, applicable only to buyers who use the wishlist feature.
- Redesign the quantity input in the checkout form to improve visibility and usability.
- Ensure smooth transitions for the Modal in the checkout process.
- Prevent users from sending empty chats; disable the send button if the chat input is empty.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-predemo-185517557`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185517557](https://www.pivotaltracker.com/story/show/185517557)

### CODE REVIEW

**Review by bergad37 — APPROVED** [review]
Well done  Test coverage has increased!!

**Review by cyubahiro367 — APPROVED** [review]

**Review by Gabriel-mdv — APPROVED** [review]
Great work!

**Review by ReneNtabana — APPROVED** [review]
LGTM

File: src/__tests__/checkout.test.jsx
Comment (irakevine) [inline src/__tests__/checkout.test.jsx:1]: It's great that you added pagination

**Review by irakevine — APPROVED** [review]
Great Work!

**Review by frankflej — APPROVED** [review]
Great!

**Review by Dyves1 — APPROVED** [review]

**Review by jcharles921 — APPROVED** [review]
Great job 👍

**Pull request merged**